ts/editor/OldEditorAdapter.svelte

```diff
@@ -29,7 +29,7 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
     const currentField = writable<EditorFieldAPI | null>(null);
 
     function updateFocus() {
-        get(activeInput)?.moveCaretToEnd();
+        /* get(activeInput)?.moveCaretToEnd(); */
```

Comment:
Seems to work well - thanks for digging into this! Just a minor point about this and the .disconnect() comment  down below. If the code or some variation of it is intended to be added back in the future, maybe a comment explaining why it's been commented out for now would be good. If the code won't be coming back, maybe it would be better to just delete it?